html_templates.go

```diff
@@ -240,7 +240,7 @@ const pingPage = `
 
      getLatencyWebSocket().then((latencies) => {
         document.getElementById('uuid').innerHTML = {{ .UUID }};
-        document.getElementById('ip').innerHTML = {{ .IPaddr }};
+        document.getElementById('ip').innerHTML = {{ .PingStats.Addr }};
         document.getElementById('values').innerHTML = latencies;
         document.getElementById('data').innerHTML = min(latencies);
         document.getElementById('icmp').innerHTML = {{ .MinIcmpRtt }};
```

Comment:
This `{{ .MinIcmpRtt }}` was part of the Results struct and could be extracted from `.PingStats`.

Reply:
Fixed in f9039f2.